spec/models/concerns/hyrax/collection_behavior_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+RSpec.describe Hyrax::CollectionBehavior, clean_repo: true do
+  let(:collection) { create(:collection_lw) }
+  let(:work) { create(:work) }
+
+  describe "#destroy" do
+    # A little bit about this spec.  It's informed by
+    # https://github.com/samvera/hyrax/issues/3872 which is why it
+    # looks a bit odd.  When I wrote this spec it worked.  I suspect
+    # that some prior change between the reported date of June 28,
+    # 2019 and today (October 8, 2020) fixed this bug.
+    #
+    # See https://github.com/samvera/hyrax/blob/0e6e7c4ff72fe508d2af0bb2eb22a1a532cf54b3/app/controllers/hyrax/batch_edits_controller.rb#L40-L47 for the related code
+    it "removes the collection id from associated members" do
+      collection.add_member_objects([work.id])
+      collection.save
+
+      collection_via_query = Hyrax.query_service.find_by_alternate_identifier(alternate_identifier: collection.id, use_valkyrie: false)
```

Comment:
better to query via ActiveFedora, since this mixin is AF specific?

Reply:
My original work had that, but I wanted it to be close to the controller logic which it was verifying.  I'm mostly ambivalent about which to use.